### PR TITLE
Make proposed changes to review process official

### PR DESCRIPTION
**Pull request by Xaeroxe**

### CODE REVIEW

**Comment by Xaeroxe** [comment]
Please note: For this PR I'm using the old rules.

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -91,10 +91,11 @@ developer, it will be merged into the source tree.
 The following rules shall be applied strictly for the `amethyst` repository. For other repositories of this organization,
 thorough review would be desirable, but no strict application is required there due to lower activity and less influence.
 
-* Pull Requests shall not be merged before at least 24 hours have passed.
 * Pull Requests shall be approved by at least two members (two approvals from contributors can count as one member approval.)
-* Merging a PR shall be done with `bors r+`
-* If a member self-requested a review, the PR shall not be merged until they reviewed the PR (exception: the member becomes inactive)
+* Pull Requests opened by organization members may be merged if approved by one member.
+* Merging a PR shall be done with `bors r+`, if possible.
```
Comment (LucioFranco) [inline docs/CONTRIBUTING.md]: Can we add more here? I would like a link to the docs and if there are more than on approver it should be `bors r=<first>, <second>,...`. We can then link to bors commands doc.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -91,10 +91,11 @@ developer, it will be merged into the source tree.
...
+* You may only block merging of a PR if the changes in it are relevant to an org team you have joined. This doesn't mean you can't
```
Comment (LucioFranco) [inline docs/CONTRIBUTING.md]: We should define what "blocked" means here. Is this a request changes?

**Pull request merged**